CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ option(ENABLE_CCACHE "Whether to use ccache to speed up compiling" ON)
 option(ENABLE_ASAN "Whether to turn AddressSanitizer ON or OFF" OFF)
 option(ENABLE_TESTING "Whether to turn unit test ON or OFF" ON)
 option(ENABLE_UBSAN "Whether to turn Undefined Behavior Sanitizer ON or OFF" OFF)
+option(ENABLE_MONITOR "Whether integrate monitor Prometheus in fact" ON)
```

Comment:
So the monitor will be active in default case ?

Reply:
Yes, default off better?